instant-distance/src/types.rs

```diff
@@ -252,6 +157,202 @@ impl PointId {
     }
 }
 
+#[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
```

Comment:
This should live near the top of the file.

Reply:
Done